main.cc

```diff
@@ -727,6 +728,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
     sharded<locator::effective_replication_map_factory> erm_factory;
     sharded<service::migration_notifier> mm_notifier;
     sharded<service::endpoint_lifecycle_notifier> lifecycle_notifier;
+    std::optional<utils::disk_space_monitor<lowres_clock>> dsm;
```

Comment:
could you please consider using a more descriptive function name like `disk_space_monitor` instead of the current abbreviated name. While the codebase currently uses abbreviations such as `cm`, `ss`, and `mm` in `main.cc`, a more comprehensive name would enhance code readability, especially for a complex function spanning over 1000 lines.

Reply:
changed to `disk_space_monitor` in v8